src/stitches.config.ts

```diff
@@ -374,6 +374,12 @@ export const dark = createTheme({
     mapNodeFade: '#FFFFFF20',
     mapLink: '#FFFFFF15',
     mapLinkDim: '#FFFFFF08',
+
+    // react-toastify
```

Comment:
comments or no comments?

Reply:
Not sure what you're asking here.  I think we should use scoped names if possible, or reuse things like $toastify-text-color-error.  As I think errorColor is too generic